src/css/carousel.scss

```diff
@@ -1,14 +1,14 @@
-.carousel-item.active,
-.carousel-item-next,
-.carousel-item-prev {
-  // this is display: block in bootstrap but we want to have cards laid out properly
-  display: flex;
-}
-
-.carousel {
+.carousel:not(.promo-carousel) {
```

Comment:
If I'm not mistaken, the key difference between the promo carousel and the other ones is that it includes the `slide` class (https://getbootstrap.com/docs/4.0/components/carousel/#slides-only). If it's true that all `slide`-type carousels should not have these styles applied, can you change this so that it's not specific to just the promo carousel?

Reply:
sure, I'll make that change